CollapseLauncher/Classes/GameManagement/GameSettings/Genshin/RegistryClass/GeneralData.cs

```diff
@@ -70,101 +107,90 @@ internal class GeneralData
 
-        public bool mtrIsOpen { get; set; } = true;
+        [JsonPropertyName("globalPerfData")]
+        public string _globalPerfData { get; set; }
+
+        //[JsonIgnore]
+        //public globalPerfData globalPerfData { get; set; }
 
         /// <summary>
-        /// This defines "<c>Vibration Intensity</c>" slider in-game. <br/>
-        /// Valid Values: 1-5
-        /// Default: 5
+        /// Something about minimap config ?
         /// </summary>
-        public int vibrationIntensity { get; set; } = 5;
-
-        public bool needPlayGoFullPkgPatch { get; set; } = false;
-        public bool exploreNotification { get; set; } = true;
-        public int mtrMaxTTL { get; set; } = 32;
-        public bool urlCheckCached { get; set; } = false;
+        public int miniMapConfig { get; set; } = 1;
 
         /// <summary>
-        /// deviceUUID<br/>
-        /// This is supposed to be empty
+        /// This  defines "<c>Automatic View Height</c>" in-game settings.<br/>
+        /// Default: true
         /// </summary>
-        public string deviceUUID { get; set; } = "";
+        public bool enableCameraSlope { get; set; } = true;
 
         /// <summary>
         /// This defines "<c>Smart combat camera</c>" whatever that is.<br/>
         /// Default: true
         /// </summary>
         public bool enableCameraCombatLock { get; set; } = true;
 
-        public bool enableEffectAssembleInEditor { get; set; } = true;
-        public bool resinNotification { get; set; } = true;
-
         /// <summary>
-        /// This define "<c>Voice Language</c>" in-game settings. <br/>
-        /// Valid values: 0-3
-        /// Default: 1
-        /// Chinese(0)
-        /// English(1)
-        /// Japanese(2)
-        /// Korean(3)
+        /// not sure either what these does.
         /// </summary>
-        public int deviceVoiceLanguageType { get; set; } = 1;
-
-        public string curAccountName { get; set; } = "";
-        public bool _audioSuccessInit { get; set; } = true;
-        public bool urlCheckAllIP { get; set; } = false;
-        public bool rewiredDisableKeyboard { get; set; } = false;
+        public bool completionPkg { get; set; } = false;
+        public bool completionPlayGoPkg { get; set; } = false;
 
         /// <summary>
-        /// This defines "<c>Audio Output</c>" combo box in-game.
-        /// Valid Values: 0 (stereo), 1 (surround)
-        /// Default: 0
+        /// Could be for PlayStation multiplayer stuff
         /// </summary>
-        public int audioOutput { get; set; } = 0;
+        public bool onlyPlayWithPSPlayer { get; set; } = false;
 
-        public int miniMapConfig { get; set; } = 1;
+        /// <summary>
+        /// Mysterious~
+        /// </summary>
+        public bool needPlayGoFullPkgPatch { get; set; } = false;
 
         /// <summary>
-        /// This  defines "<c>Automatic View Height</c>" in-game settings.<br/>
-        /// Default: true
+        /// Mobile notification stuff
```

Comment:
Maybe we should elaborate more as to what "stuff" actually means in this context, for future reference.

Reply:
Made it clearer as of https://github.com/neon-nyan/Collapse/commit/f2f38dd39f11b69fdc06ac1e2bca0530f6571b40